scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -171,32 +275,26 @@ library_ids <- c(
   "SCPCL000491"
 )
 
-# subset sce to only contain samples with library IDs of interest 
+# subset sce to only contain samples with IDs of interest 
 samples_to_keep <- integrated_sce$sample %in% library_ids
 rms_sce <- integrated_sce[,samples_to_keep]
 
-# remove original integrated object 
+# remove original integrated object to save some memory
```

Comment:
Oh boy I read this as save money (even though this was originally my comment) and I was like, "wow, does this really affect server cost!? I'm surprised!"
Having fun!